src/sql-parser/tests/testdata/scalar

```diff
@@ -463,6 +554,57 @@ parse-scalar
 ----
 Op { op: Op { namespace: [], op: "+" }, expr1: Subquery(Query { ctes: Simple([]), body: Select(Select { distinct: None, projection: [Expr { expr: Value(Number("1")), alias: None }], from: [], selection: None, group_by: [], having: None, options: [] }), order_by: [], limit: None, offset: None }), expr2: Some(Subquery(Query { ctes: Simple([]), body: Select(Select { distinct: None, projection: [Expr { expr: Value(Number("2")), alias: None }], from: [], selection: None, group_by: [], having: None, options: [] }), order_by: [], limit: None, offset: None })) }
 
+parse-scalar roundtrip
+1 < 2
+----
+1 < 2
+
+parse-scalar roundtrip
+1 <= 2
+----
+1 <= 2
+
+parse-scalar roundtrip
+1 >= 2
+----
+1 >= 2
+
+parse-scalar roundtrip
+1 > 2
+----
+1 > 2
+
+parse-scalar roundtrip
+1 - 2
+----
+1 - 2
+
+parse-scalar roundtrip
+1 * 2
+----
+1 * 2
+
+parse-scalar roundtrip
+1 % 2
+----
+1 % 2
+
+parse-scalar roundtrip
+1 / 2
+----
+1 / 2
+
+# TODO(def-) Enable when fixed
```

Comment:
Same here, was hoping for this to be fixed, but we can also handle it as a follow-up and get the rest in, which will already be a definitive improvement.